.github/workflows/benchmark.yml

```diff
@@ -109,7 +109,11 @@ jobs:
         run: |
           docker run --name scylladb -p 9042:9042 -p 19042:19042 -p 9160:9160 -p 7000:7000 -p 7001:7001 -p 7199:7199 -p 9180:9180 -d scylladb/scylla:latest --broadcast-address 127.0.0.1 --listen-address 0.0.0.0 --broadcast-rpc-address 127.0.0.1
           sleep 15 # Wait for ScyllaDb to initialize
-
+      - name: Install SurrealDB
```

Comment:
There exists a surrealdb module in testcontainers-go. I think we must use it here